### PR TITLE
Disallow serializing and transferring non-origin-clean ImageBitmap

### PR DESCRIPTION
**Pull request by yutakahirano**
The tests update for https://github.com/whatwg/html/pull/5114.
Add one test case for serialization, add one test case for
transferring. The relationship between COOP/COEP is now clear
and we can remove no-coop-coep.https.window.js.

### CODE REVIEW

**Comment by yutakahirano** [comment]
@domenic, can you take a look at this change too?

File: 2dcontext/imagebitmap/createImageBitmap-serializable.html [hunk 1]
```diff
@@ -40,4 +41,12 @@
...
     }, `Serialize ImageBitmap created from ${name}`);
 }
+
+promise_test(async (t) => {
+  const url = get_host_info().REMOTE_ORIGIN + '/images/pattern.png';
+  const image = await makeMakeHTMLImage(url)();
+  const bitmap = await createImageBitmap(image);
+
+  assert_throws('DataCloneError', () => worker.postMessage(bitmap));
+}, 'Serializing a non-origin-clean ImageBitmal throws.');
```
Comment (domenic) [inline 2dcontext/imagebitmap/createImageBitmap-serializable.html]:
```suggestion
}, 'Serializing a non-origin-clean ImageBitmap throws.');
```

File: 2dcontext/imagebitmap/createImageBitmap-transfer.html [hunk 2]
```diff
@@ -39,4 +40,14 @@
...
 }
+
+promise_test(async (t) => {
+  const url = get_host_info().REMOTE_ORIGIN + '/images/pattern.png';
+  const image = await makeMakeHTMLImage(url)();
+  const bitmap = await createImageBitmap(image);
+
+  assert_throws('DataCloneError',
+                () => worker.postMessage(bitmap, [bitmap]));
+}, 'Transferring a non-origin-clean ImageBitmal throws.');
```
Comment (domenic) [inline 2dcontext/imagebitmap/createImageBitmap-transfer.html]:
```suggestion
}, 'Transferring a non-origin-clean ImageBitmap throws.');
```

**Review by domenic — APPROVED** [review]
LGTM with nits. Try rebasing on latest master and force-pushing to see if we can fix the CI status? Otherwise we will need to ask for admin merge.

File: 2dcontext/imagebitmap/createImageBitmap-serializable.html (hunk 1 above)
Comment (yutakahirano) [reply 2dcontext/imagebitmap/createImageBitmap-serializable.html]: Done.

File: 2dcontext/imagebitmap/createImageBitmap-transfer.html (hunk 2 above)
Comment (yutakahirano) [reply 2dcontext/imagebitmap/createImageBitmap-transfer.html]: Done.

**Comment by yutakahirano** [comment]
Thank you!

**Pull request merged**